invenio_app_ils/records/resolver/jsonresolver/document_item.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Resolve the EItem referenced in the Document."""
+
+import jsonresolver
+from werkzeug.routing import Rule
+
+from invenio_app_ils.search.api import ItemSearch
+
+
+@jsonresolver.hookimpl
+def jsonresolver_loader(url_map):
+    """Resolve the referred EItems for a Document record."""
+    from flask import current_app
+
+    def items_resolver(document_pid):
+        """Search and return the total number of items."""
+        # NOTE: in comparison to eitems we don't return the actual items in
```

Comment:
Actually, I was wrong, there should be many. It is Documents attached to Serial that can be too many.
It should be safe to resolve items here.
To add it with a few fields.